hack/test-extended/default/run.sh

```diff
@@ -39,8 +39,9 @@ set +e
 
 # Compile the extended tests first to avoid waiting for OpenShift server to
 # start and fail sooner on compilation errors.
+echo "[INFO] Compiling test/extended package ..."
 GOPATH="${OS_ROOT}/Godeps/_workspace:${GOPATH}" \
-  ginkgo build -r ./test/extended 
+  go test -c ./test/extended -o ${OS_OUTPUT_BINPATH}/extended.test || exit 1
```

Comment:
@deads2k I moved this up so it fails sooner if you make a typo when writing extended tests... waiting ~2min for openshift to start to realize that you forgot to close string is frustrating ;D